zend_abstract_interface/config/config_stable_file.c

```diff
@@ -0,0 +1,112 @@
+#include "../tsrmls_cache.h"
+
+#include <components-rs/library-config.h>
+
+#include "config.h"
+#include "config_stable_file.h"
+
+HashTable *config_values = NULL;
+
+bool zai_config_stable_file_get_value(zai_str name, zai_env_buffer buf) {
+    if (!config_values) {
+        return false;
+    }
+
+    zval *value = zend_hash_str_find(config_values, name.ptr, name.len);
```

Comment:
```suggestion
    zval *value = zend_hash_find(config_values, name);
```
Heh, now we have the persistent string, then let's also use it :-D

Reply:
`name` is a `zai_str`